### DISCUSSION TITLE
how to make OPYN as great defi options market?

### DISCUSSION THREAD

**Discussion by jayaworld2002** [opynfinance/OGP#6] [General]
how to make OPYN as great defi options market? 

Please provide your ideas and we can discuss.

Similar to including gelato network codes. or uniswap V3 features. We are provide innovative ideas or existing options markets features integration also ok.

Instead of continuing with ETH network(rising  ETH price and sky rocketing GAS prices), changing to other cheaper networks like BSC...?

**Comment by R3dCr0ss** [comment]
hmm
